### PR TITLE
Create new endpoint for getting orphan disks

### PR DESCRIPTION
**Pull request by barthy1**
* new endpoint /orphan_disks
* leave existing logic with endpoint /disks to be compatible with older versions

[#145243179](https://www.pivotaltracker.com/story/show/145243179)

Signed-off-by: Konstantin Maksimov <kmaksimov@ru.ibm.com>

### CODE REVIEW

File: src/bosh-director/lib/bosh/director/api/controllers/orphan_disks_controller.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+require 'bosh/director/api/controllers/base_controller'
+
+module Bosh::Director
+  module Api::Controllers
+    class OrphanDisksController < BaseController
+      get '/' do
+        content_type(:json)
+        orphan_json = OrphanDiskManager.new(@logger).list_orphan_disks
```
Comment (dpb587-pivotal) [inline src/bosh-director/lib/bosh/director/api/controllers/orphan_disks_controller.rb]: Traditionally we try to create these managers once in the `initialize`, then in the function invoke the desired method ([example](https://github.com/barthy1/bosh/blob/c85667afea2e1e12b06240542243a3d49f68004a/src/bosh-director/lib/bosh/director/api/controllers/deployments_controller.rb#L43)). Not critical, but helps us keep our `new`ers in a consistent place until we can do better at DI.

File: src/bosh-director/lib/bosh/director/api/controllers/orphan_disks_controller.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+require 'bosh/director/api/controllers/base_controller'
+
+module Bosh::Director
+  module Api::Controllers
+    class OrphanDisksController < BaseController
```
Comment (dpb587-pivotal) [inline src/bosh-director/lib/bosh/director/api/controllers/orphan_disks_controller.rb]: For consistency with the route, can we rename this class/file to be "orphaned".

**Comment by cppforlife** [comment]
i think we ve named all are things "orphan" so route should be "/orphan_disks".

**Comment by dpb587-pivotal** [comment]
I liked how the story title said `GET /orphan_disks` and the description said `GET /orphaned_disks` - I couldn't decide which was correct :)

So I guess it's a route rename, then; not a class/file rename.

**Pull request merged**